### PR TITLE
[Docs] Fix some experimental/deprecated hints

### PR DESCRIPTION
**Pull request by AThousandShips**
Two cases from the recent rendering PR that went unnoticed, and some cases that have a deprecated or experimental message but doesn't use the property

Can remove some of those if they are incorrect, but I think they match, if they have the message in the description I don't think there's an issue making it an "official" tag

~Left some of the cases in the Android export in a separate commit for simplicity, as they are specific to an API level, will restore them if they're unwanted~

### CODE REVIEW

**Review by m4gr3d — APPROVED** [review]
The Android updates look good!

**Comment by Mickeon** [comment]
For all of these, there's no need to repeat "This is experimental", and explain what "experimental" means because the mere existence of the message infers it.

**Comment by AThousandShips** [comment]
Can reword the ones that do slightly :)

File: doc/classes/NavigationRegion2D.xml [hunk 1]
```diff
@@ -89,9 +89,8 @@
 		<member name="avoidance_layers" type="int" setter="set_avoidance_layers" getter="get_avoidance_layers" default="1">
 			A bitfield determining all avoidance layers for the avoidance constrain.
 		</member>
-		<member name="constrain_avoidance" type="bool" setter="set_constrain_avoidance" getter="get_constrain_avoidance" default="false">
+		<member name="constrain_avoidance" type="bool" setter="set_constrain_avoidance" getter="get_constrain_avoidance" default="false" experimental="Should not be used in production as agent&apos;s can get stuck on the navigation polygon corners and edges especially at high frame rate.">
```
[the remark this answers is not in the document]
Comment (AThousandShips) [reply doc/classes/NavigationRegion2D.xml:92]: Production is an established term for releases (see [here](https://docs.godotengine.org/en/stable/tutorials/editor/using_the_web_editor.html#using-the-web-editor), [here](https://docs.godotengine.org/en/stable/tutorials/scripting/gdextension/what_is_gdextension.html#supported-languages), [here](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_documentation_comments.html?highlight=Production%20#deprecated-and-experimental-tags), etc.), it's even used for a build option, so I'll keep that wording, like so
```suggestion
	<member name="constrain_avoidance" type="bool" setter="set_constrain_avoidance" getter="get_constrain_avoidance" default="false" experimental="When enabled, agents are known to get stuck on the navigation polygon corners and edges, especially at a high frame rate. Not recommended for use in production at this stage.">
```

File: doc/classes/ProjectSettings.xml [hunk 2]
```diff
@@ -2345,9 +2345,8 @@
 			If [code]true[/code], performs a previous depth pass before rendering 3D materials. This increases performance significantly in scenes with high overdraw, when complex materials and lighting are used. However, in scenes with few occluded surfaces, the depth prepass may reduce performance. If your game is viewed from a fixed angle that makes it easy to avoid overdraw (such as top-down or side-scrolling perspective), consider disabling the depth prepass to improve performance. This setting can be changed at run-time to optimize performance depending on the scene currently being viewed.
 			[b]Note:[/b] Depth prepass is only supported when using the Forward+ or Compatibility rendering method. When using the Mobile rendering method, there is no depth prepass performed.
 		</member>
-		<member name="rendering/driver/threads/thread_model" type="int" setter="" getter="" default="1">
+		<member name="rendering/driver/threads/thread_model" type="int" setter="" getter="" default="1" experimental="This option has several known bugs which can lead to crashing, especially when using particles or resizing the window. Not recommended for use in production at this stage.">
```
[the remark this answers is not in the document]
Comment (AThousandShips) [reply doc/classes/ProjectSettings.xml:2348]:
```suggestion
		<member name="rendering/driver/threads/thread_model" type="int" setter="" getter="" default="1" experimental="This setting has several known bugs which can lead to crashing, especially when using particles or resizing the window. Not recommended for use in production at this stage.">
```

**Comment by AThousandShips** [comment]
Thank you! Added these changes

**Review by Mickeon — APPROVED** [review]
Surely more of these won't pop up again now, _right_?

**Comment by AThousandShips** [comment]
There are no uses of "experimental" outside of these tags now in my search, or "deprecated"

**Comment by Mickeon** [comment]
Yeah but as this PR needed to happen there may be some others that may be merged without noticing the new `deprecated`/`experimental` message.

**Comment by AThousandShips** [comment]
That's true, but keeping an eye for those, but good point

**Comment by AThousandShips** [comment]
Thank *you*!

**Pull request merged**